plugins/woocommerce/client/blocks/assets/js/blocks/cart-checkout-shared/payment-methods/payment-methods.tsx

```diff
@@ -76,9 +78,8 @@ const PaymentMethods = ( {
 					) }
 					wrapperElement="p"
 					wrapperProps={ {
-						className: [
+						className:
```

Comment:
It's passed to `clsx` later; it doesn't have to be an array (and it produces a TypeScript error).